src/atomic.dex.app.cpp

```diff
@@ -908,7 +909,17 @@ namespace atomic_dex
             }
 
             auto tx_fee_value     = QString::fromStdString(get_formated_float(tx_fee_f));
-            auto final_balance    = get_formated_float(t_float_50(amount.toStdString()) - (trade_fee_f + tx_fee_f));
+            auto final_balance_f  = t_float_50(amount.toStdString()) - (trade_fee_f + tx_fee_f);
+            std::string final_balance = amount.toStdString();
+            if (final_balance_f > 0)
+            {
+                final_balance = get_formated_float(final_balance_f);
+                out.insert("not_enough_balance_to_pay_the_fees", false);
+            } else {
+                out.insert("not_enough_balance_to_pay_the_fees", true);
+                auto fees_needed = t_float_50(0.00777) - final_balance_f;
```

Comment:
TODO: tagging this dirty "hackarounds" for improvement in next iteration - ref. discussion with @Milerius

Reply:
Will introduce a constant value function into the next iteration to solve this.

cc @naezith .